src/shared/components/column-select/ColumnSelect.tsx

```diff
@@ -33,6 +32,7 @@ import {
   ColumnSelectTab,
 } from '../../../uniprotkb/types/resultsTypes';
 
+import '../../styles/sticky.scss';
```

Comment:
Could use css modules while we're at it?

Reply:
Done now